packages/botonic-nlp/src/tasks/text-classification/storage/config-storage.ts

```diff
@@ -0,0 +1,17 @@
+import { mkdirSync, writeFileSync } from 'fs'
+import { join } from 'path'
+
+import { CONFIG_FILENAME } from '../../../storage/constants'
```

Comment:
personally I prefer to put the constants as close as possible to where they're used (so here it would be within TextClassificationConfigStorage (and in private const), since no other class will ever need to know about it)